src/MinimumNodeService.cpp

```diff
@@ -27,14 +27,19 @@ void MinimumNodeService::initSetupFromUninitialised()
 {
   // DEBUG_SERIAL << F("> initiating Normal negotation") << endl;
 
+  // enumerate the CAN bus to allocate a free CAN ID
+  Action action = {ACT_START_CAN_ENUMERATION, false };
+  controller->putAction(action);
+
+  initSetupCommon();
+}
+
+void MinimumNodeService::initSetupCommon()
```

Comment:
The bits here are also used by initSetupFromNormal() below.

Reply:
I agree.  Good catch!